web/views.py

```
@@ -3788,6 +3793,154 @@ def get_object(self):
+            base_slug = slugify(meetup.title)
+            slug = base_slug
+            counter = 1
+
+            while Meetup.objects.filter(slug=slug).exists():
+                slug = f"{base_slug}-{counter}"
+                counter += 1
+
+            meetup.slug = slug
+            meetup.save()
+            return redirect("meetup_list")
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)